Hide group title in `CrudMoreActionsMenu` when only one group is present (#2920)
vivid-planet/comet: .changeset/cyan-shoes-beg.md

> ---
> "@comet/admin": minor
> ---
> 
> Update `CrudMoreActionsMenu` Design
> 
> -   remove `groupTitle` if only one action group is present

Reviewer: ```suggestion
---
"@comet/admin": minor
---

Hide group title in `CrudMoreActionsMenu` when only one group is present
```
